ice/emulators.py

```diff
@@ -30,12 +29,14 @@ def emulator_rom_launch_command(emulator, rom):
   # substitute values in at runtime. Right now the only supported values are:
   # %l - The location of the emulator (to avoid sync bugs)
   # %r - The location of the ROM (so the emulator knows what to launch)
+  # %n - The ROM filename without its extension (for emulators that utilize separete configuration files)
```

Comment:
Can you turn this into `%fn` instead? ROMs already have the concept of a `name`, which isn't the same as the filename (see (rom_parser_tests)[https://github.com/scottrice/Ice/blob/master/tests/rom_parser_tests.py] for examples of how the filename and the name can be different).

I can't tell if what you need for your use case is filename or name (seems like either would most likely work) so its just a matter of which one you want to use. With the current implementation I would use the `%fn` modifier, or if you want to use `%n` you should pass it `rom.name`